### ISSUE TITLE
object 'dataset_size' not found

### ISSUE THREAD

**Issue by stanley-rudkin-moj** [kvasilopoulos/onsr#5]
Hi there, I am finding that if I run the code below I get the error `Error in restrict_size(1, "Gb$") : object 'dataset_size' not found`. Apologies if this is not a bug and just my mistake!

```r
library(onsr)
ons_get(id="mid-year-pop-est")
```

**Comment by kvasilopoulos** [comment]
Hi @stanley-rudkin-moj can you please make sure that you have the latest version? I think this was an issue that has been resolved in `1.0.1`

```
> packageVersion("onsr")
[1] ‘1.0.1’
```
If not please remove and reinstall.

**Comment by stanley-rudkin-moj** [comment]
Thank you! I was using `1.0.0`.
